### DISCUSSION TITLE
Any idea what I'm doing wrong here?

### DISCUSSION THREAD

**Discussion by ORi0N** [jomjol/AI-on-the-edge-device#601] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/701149/150699940-acf1a862-c340-48d3-951c-c1be643d012a.png)

Can't make the letters any more clear actually? Any tips?
Could I train the OCR for black on white perhaps?

**Comment by jomjol** [comment, marked as answer]
Firstly you can improve the ROI settings according to this documentation in the wiki: https://github.com/jomjol/AI-on-the-edge-device/wiki/ROI-Configuration

**Reply by ORi0N** [reply]
Awesome. Why did I miss that? Perhaps could be a good idea to add some small pointers on the Digital ROI page itself with "good" and "bad" example (just like on the wiki page) as some quick pointers for people to notice.

**Comment by jomjol** [comment]
There were some times, where the features developed faster than the documenation. As I'm still the only one who is developing this, there will always be a gap. This could only be solved by a commercial variant, which I'm not following.

Best regards,
   jomjol
